domapping/cli.py

```diff
@@ -81,14 +89,23 @@ def schema_to_mapping_cli(schema, output, config, indent):
 @click.argument('output', type=click.File('w'), default='-')
 @click.option('--indent', '-i', default=4, type=click.INT,
               help='Output template indentation step.')
-@click.option('--type', '-t', default='type', type=click.STRING,
+@click.option('--type', '-t', type=click.STRING,
```

Comment:
I know I am the one who put suggested `es_type` as the option name for `schema_to_mapping` but now we have an inconsistency between `--type` and `--es_type` which designate the same thing. We should choose one of them and use the option name for both commands. `--type` is shorter but at the same time it is confusing as `schema_to_mapping_cli ` also works with JSON Schema types, so I would go for using `--es_type` in `mapping_to_jinja_cli` too. What do you think of it?